doc/ManifestSpecv1.0.md

```diff
@@ -0,0 +1,168 @@
+   2) YAML Syntax
+   3) Minimal singleton YAML file example
+   4) Multiple file YAML example
+   5) Best Practices
+
+# YAML Specification
+
+## YAML file name and folder structure
+YAML files shall be added to the repository with the following folder structure:
+manifests / p / publisher / package / packageVersion / publisher.package.manifestFile.yaml
+
+Example:
+manifests / m / Microsoft / WindowsTerminal / 1.6.10571.0 / WindowsTerminal.yaml
+
+* Manifests are partitioned by the first letter of the publisher name (in lower case). For example: m.
+* Publisher folder is the name of the company that publishes the tool.  For example: Microsoft.
+* The child folder package is the name of the application or tool.  For example: WindowsTerminal.
+* The child folder package version is the version of the package. For example: 1.6.10571.0.
+* The filename must be a combination of the application name and the current version.  For example: WindowsTerminal.yaml.
+
+The publisher and application folders MUST match the values used to define the Id.  See PackageIdentifier: in the YAML for more detail.
+The version in the folder name MUST match the version field value in the YAML file.  See PackageVersion: in the YAML for more detail.
+
+There are two primary types of manifests. A single file manifest (singleton) and a multi-file manifest. 
+[JSON schemas](https://github.com/microsoft/winget-cli/tree/master/schemas/JSON/manifests/v1.0.0) have been provided 
+to help strongly type attributes and requirements.
+
+## YAML Syntax
+Each field in the file must be PascalCased and cannot be duplicated.
+
+## Minimal singleton YAML file example
+As specified in the [singleton JSON schema](https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.singleton.1.0.0.json),
+only a number of fields are required.  The minimal supported YAML file would look like the example below. The singleton format is only valid for packages containing
+a single installer and a single locale. If more than one installer or locale is provided, the multiple YAML file format and schema must be used.
+
+The partitioning scheme was added to help with GitHub's UX. Folders with thousands of children do not render well in the browser.
+
+Path: manifests / m / Microsoft / WindowsTerminal / 1.6.10571.0 / WindowsTerminal.yaml
+
+```YAML
+PackageIdentifier: "Microsoft.WindowsTerminal"
+PackageVersion: "1.6.10571.0"
+PackageLocale: "en-US"
+Publisher: "Microsoft"
+PackageName: "Windows Terminal"
+License: "MIT"
+ShortDescription: "The new Windows Terminal, a tabbed command line experience for Windows."
+Installers: 
+ - Architecture: "x64"
+   InstallerUrl: "https://github.com/microsoft/terminal/releases/download/v1.6.10571.0/Microsoft.WindowsTerminal_1.6.10571.0_8wekyb3d8bbwe.msixbundle"
+   InstallerSha256: 092aa89b1881e058d31b1a8d88f31bb298b5810afbba25c5cb341cfa4904d843
+   SignatureSha256: e53f48473621390c8243ada6345826af7c713cf1f4bbbf0d030599d1e4c175ee
```

Comment:
Is InstallerType no longer required?

Reply:
Great catch! I've been working on this one so much I missed it.